mkt/ratings/tests/test_views.py

```diff
@@ -266,8 +266,7 @@ def test_filter_by_nonpublic_app(self, get_region_mock):
         get_region_mock.return_value = mkt.regions.USA
         res, data = self._get_filter(
             app=self.app.app_slug, expected_status=403)
-        eq_(data['detail'], 'The app requested is not public or not available '
-                            'in region "us".')
+        eq_(data['detail'], 'The app requested is not public "us".')
```

Comment:
Its the part of validation. As in line 266 its getting the value for region USA that is being compared with the error detail. If I will remove "us", test case will be failed.